### DISCUSSION TITLE
How to enter the administrator interface

### DISCUSSION THREAD

**Discussion by KKISO574** [JulianPrieber/littlelink-custom#450] [General]
I can't find my admin background interface. I'm using docker deployment. How can I return to the administrator page?

**Comment by lastsamurai26** [comment]
Which reverse proxy are you using? 
What does your config look like?

in the normal you have the main page for logging in and there you will also find the admin panel provided that the main page has not been changed, otherwise you must call the admin panel directly with /dashboard

**Comment by altinturk** [comment]
I had the same question on the self hosted version; after the installation, how do you go to the admin panel? There are no cluses on the documentation. 
Then i downloaded docker and realized the admin page is actually behind /dashboard path in the url.

At least on documentation side, at post-install, please explicitly mention that type of information.
